src/map/CustomDriftMarker.tsx

```diff
@@ -0,0 +1,50 @@
+import {Rectangle, Tooltip} from "react-leaflet";
+import React, {useState} from "react";
+import {DriftMarker} from "leaflet-drift-marker";
+
+interface BucketObject {
+  ltAvg: number,
+  lnAvg: number,
+  val: number,
+  count: number,
+  ltMin: number,
+  ltMax: number,
+  lnMin: number,
+  lnMax: number
+}
+
+interface CustomDriftMarkerProps {
+  bucket: BucketObject,
```

Comment:
Seems much cleaner with this approach :-)

I think a `bounds` prop would be more general than the `bucket` prop (which has quite a lot of back-end specific property names)

Reply:
I could pass just a bounds prop but I'd also need to bring in ltAvg, lnAvg, val, count. Would you rather I pass those through explicitly than pass the whole 'bucket'?